### ISSUE TITLE
[DataCap Application] Guangdong Zhiyun Urban Construction Technology Co., Ltd.

### ISSUE THREAD

**Issue by tanjian917** [filecoin-project/filecoin-plus-large-datasets#2184]
### Data Owner Name

Guangdong Zhiyun Urban Construction Technology Co., Ltd.

### What is your role related to the dataset

Dataset Owner

### Data Owner Country/Region

China

### Data Owner Industry

Life Science / Healthcare

### Website

https://www.jiancaiyi.com/

### Social Media

```text
https://www.jiancaiyi.com/
```

### Total amount of DataCap being requested

10PiB

### Expected size of single dataset (one copy)

1024

### Number of replicas to store

10

### Weekly allocation of DataCap requested

1PiB

### On-chain address for first allocation

f1zvurht6mq5qbo6gq7r7wxisw25z262ocmbg7pva

### Data Type of Application

Public, Open Commercial/Enterprise

### Custom multisig

- [ ] Use Custom Multisig

### Share a brief history of your project and organization

```text
Guangdong Zhiyun Urban Construction Technology Co., Ltd. is the largest architectural design company in South China that integrates raw material production, sales, construction, design, decoration, and after-sales. It is a digital solution provider for the entire construction industry. We have gathered thousands of construction materials construction companies. There are hundreds of architectural designers providing services to customers. From raw materials to construction to after-sales, we have everything.
```

### Is this project associated with other projects/ecosystem stakeholders?

No

### Describe the data being stored onto Filecoin

```text
Guangdong Zhiyun Urban Construction Technology Co., Ltd. is the largest architectural design company in South China that integrates raw material production.As an architectural design company that integrates raw material production, sales, construction, design, decoration, and after-sales, we have a wide variety of data.
Including construction equipment, architectural design manuscripts, CAD drawings, construction pictures and videos, customer negotiations, etc.
```

### Where was the data currently stored in this dataset sourced from

My Own Storage Infra

### If you are a data preparer. What is your location (Country/Region)

China

### Please share a sample of the data

```text
As an architectural design company that integrates raw material production, sales, construction, design, decoration, and after-sales, we have a wide variety of data.
Link: https://pan.baidu.com/s/1IGhv0Cb-k9LKmQtkXTsOtQ?pwd=6vjh Extraction code: 6vjh
Link: https://pan.baidu.com/s/1shij_nj3vXTIzXzzLoBblQ?pwd=3f41 Extraction code: 3f41
Link: https://pan.baidu.com/s/1PUGbZh1JUPTlb3NLBHig4Q?pwd=vjdy Extraction code: vjdy
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [X] I confirm

### What is the expected retrieval frequency for this data

Monthly

### For how long do you plan to keep this dataset stored on Filecoin

1.5 to 2 years

### In which geographies do you plan on making storage deals

Greater China, Asia other than Greater China, North America, Europe

### How will you be distributing your data to storage providers

Cloud storage (i.e. S3), HTTP or FTP server, IPFS, Shipping hard drives, Lotus built-in data transfer

### How do you plan to choose storage providers

Slack, Filmine

### How do you plan to make deals to your storage providers

Boost client, Lotus client, Bidbot

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by Sunnyiscoming** [comment]
- Have you prepared enough token for sector pledge?
- Are you a data preparer? What is your previous experience as a data-preparer? List previous applications and client IDs
- How will the data be prepared? Please include tooling used and technical details
- If you are not preparing the data, who will prepare the data?  (Name and Business)
- Has this dataset been stored on Filecoin before? If so, why are you choosing to store it again?
- Best practice for storing large datasets includes ideally, storing it in 3 or more regions, with 4 or more storage provider operators or owners.You should list Miner ID, Business Entity, Location of sps you will cooperate with.
- Per the https://github.com/filecoin-project/notary-governance/issues/922 for Open, Public Dataset applicants, please complete the following [Fil+ registration form](https://form.jotform.com/231862753015656) to identify yourself as the applicant and also please add the contact information of the SP entities you are working with to store copies of the data.
- Could you send an email to filplus-app-review@fil.org with your official domain in order to confirm your identity? Email name should includes the issue id #2184.

**Comment by tanjian917** [comment]
Hey, great to see your reply.
1. The token for sector pledge is sufficient and we can start it immediately.
2. I am the owner of the data and also the preparer of the data. In order to verify our identity, I have sent an email through the domain name mailbox. Please check it.
3. We have contacted many SPs, and we temporarily decided to cooperate with these SPs first. Here are some SPs. We are trying to find more SPs:
f02221112: Guangzhou
f01920887: Australia
f02246008: Hong Kong
f02199203: Inner Mongolia
f02370792: Guangdong
f02200472: Sichuan
![WechatIMG1090](https://github.com/filecoin-project/filecoin-plus-large-datasets/assets/42170625/44c9779c-e498-4800-821d-59e79fc49451)

**Comment by tanjian917** [comment]
Looking forward to seeing you agree us, best wishes to you!

**Comment by herrehesse** [comment]
@tanjian917 

What are their business names and are they on VPN?

- f02221112: Guangzhou
- f01920887: Australia
- f02246008: Hong Kong
- f02199203: Inner Mongolia
- f02370792: Guangdong
- f02200472: Sichuan

**Comment by tanjian917** [comment]
Thank you for your reminder, we added the company name, SP did not use VPN, we are honest, thank you!
f02221112: GuangDong    yuanxing
f01920887: Australia        Sindurot
f02246008: Hong Kong      Calculus
f02199203: Inner Mongolia   KuQi 
f02370792: GuangDong     Star Alliance 
f02200472: Sichuan     longcould

**Comment by Sunnyiscoming** [comment]
Have you completed  [Fil+ registration form](https://form.jotform.com/231862753015656)?

**Comment by herrehesse** [comment]
<img width="1197" alt="Screenshot 2023-08-31 at 17 03 14" src="https://github.com/filecoin-project/filecoin-plus-large-datasets/assets/70132857/1d2c64a2-ba5a-4a13-a8ef-67874c60a65a">

Here are previous results of the same mining group. Seems to me that these miners belong to the same entity as they are noted by multiple KYC applicants as the same organisation. Most of them seem reachable and no previous VPN history.

**Comment by tanjian917** [comment]
Hey, brother, in your picture, there are other LDN SPs, which are basically completely different from the SPs we listed.

> <img alt="截图 2023-08-31 于 17 03 14" width="1197" src="https://user-images.githubusercontent.com/70132857/264688735-1d2c64a2-ba5a-4a13-a8ef-67874c60a65a.png">
> 以下是同一采矿组之前的结果。在我看来，这些矿工属于同一实体，因为多个 KYC 申请人将它们视为同一组织。其中大多数似乎可以访问，并且以前没有 VPN 历史记录。

**Comment by tanjian917** [comment]
Yes, we submitted it, thank you @Sunnyiscoming 
<img width="825" alt="WX20230901-180952@2x" src="https://github.com/filecoin-project/filecoin-plus-large-datasets/assets/42170625/6a37cdf1-08e7-48ad-af6b-176cc66f974a">

**Comment by Sunnyiscoming** [comment]
### Datacap Request Trigger
**Total DataCap requested**
> 10PiB

**Expected weekly DataCap usage rate**
> 1PiB

**Client address**
>f1zvurht6mq5qbo6gq7r7wxisw25z262ocmbg7pva

**Comment by Sunnyiscoming** [comment]
Please continue to do investigation about sp.

**Comment by herrehesse** [comment]
@Sunnyiscoming why do you open up a trench and then start to investigate?

**Comment by Bitengine-reeta** [comment]
````
$ lotus net connect f02221112
f02221112 -> {12D3KooWFoGjYwLmkzpDAnANZfBE5fRM7XDjxLeqn5ewhfaoiCqm: [/ip4/[14.119.112.16/tcp/21341](http://14.119.112.16/tcp/21341)]}
ERROR: failed to parse multiaddr "f02221112": must begin with /

$ lotus net connect f01920887
f01920887 -> {12D3KooWSWpG2Rsbwr2J7c7omDVuiw6BPBZwBMSrSHjpjfWa4GD2: [/ip4/[203.129.131.94/tcp/11288](http://203.129.131.94/tcp/11288)]}
ERROR: failed to parse multiaddr "f01920887": must begin with /

$ lotus net connect f02246008
f02246008 -> {12D3KooWQA1VguWa7de3ajNf2sQ5i7feaiKppjsjs12b8Qggj1cn: [/ip4/[116.92.243.2/tcp/24002](http://116.92.243.2/tcp/24002)]}
ERROR: failed to parse multiaddr "f02246008": must begin with /

$ lotus net connect f02199203
f02199203 -> {12D3KooWLUiX6qKj1RXHWdHGBifKPwT1DQ1UdAoTLAskC82mUTUS: [/ip4/[116.136.130.130/tcp/6789](http://116.136.130.130/tcp/6789)]}
ERROR: failed to parse multiaddr "f02199203": must begin with /

$ lotus net connect f02370792
f02370792 -> {12D3KooWKa1NQKxveFmppQAjkHG8RGpjoQ2ivubeNuEUgZGgc2T6: [/ip4/[119.147.160.231/tcp/41555](http://119.147.160.231/tcp/41555)]}
ERROR: failed to parse multiaddr "f02370792": must begin with /

$ lotus net connect f02200472
f02200472 -> {12D3KooWDeJTed3dzgLYEAHsfLcjCuBSCHj1XYzJppA9phf4kLbZ: [/ip4/[171.93.112.10/tcp/41335](http://171.93.112.10/tcp/41335)]}
ERROR: failed to parse multiaddr "f02200472": must begin with /
````

Checked all the MinerIDs provided by the client are reachable and use different IPs.
In addition, the client has completed the KYC email certification and registration form submission. I will provide support in the first round and stay tuned.

**Comment by Bitengine-reeta** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacecmgwx3xv4k7sf4gvfikdwwr3zyawfyp52aouj4vpjh2d7mhfjvze 
#### Address 
> f1zvurht6mq5qbo6gq7r7wxisw25z262ocmbg7pva
#### Datacap Allocated
> 512.00TiB
#### Signer Address
> f1jyvhxp4kmwreo22ke4itspraznpudw3uqaink5i
#### Id
> e6bae94c-e68e-4e9e-8e69-acbf599210fe
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacecmgwx3xv4k7sf4gvfikdwwr3zyawfyp52aouj4vpjh2d7mhfjvze

**Comment by Tom-OriginStorage** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzaceakzk2kqhkctwyty72cgrb4y26hyorqgf2ivaqfiw4buiejcb3nzu 
#### Address 
> f1zvurht6mq5qbo6gq7r7wxisw25z262ocmbg7pva
#### Datacap Allocated
> 512.00TiB
#### Signer Address
> f1q6bpjlqia6iemqbrdaxr2uehrhpvoju3qh4lpga
#### Id
> e6bae94c-e68e-4e9e-8e69-acbf599210fe
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaceakzk2kqhkctwyty72cgrb4y26hyorqgf2ivaqfiw4buiejcb3nzu

**Comment by herrehesse** [comment]
@Bitengine-reeta & @Tom-OriginStorage - Highly questionable signatures, if the data or retrieval is not in order will instantly dispute.

**Comment by tanjian917** [comment]
checker:manualTrigger

**Comment by tanjian917** [comment]
checker:manualTrigger

**Comment by tanjian917** [comment]
checker:manualTrigger

**Comment by tanjian917** [comment]
The data has not been updated, the retrieval data is incorrect,  the SP has not received the bot's retrieval record. Please help to check it. @raghavrmadya @Sunnyiscoming @simonkim0515 @fabriziogianni7 @jbesraa

**Comment by tanjian917** [comment]
checker:manualTrigger

**Comment by tanjian917** [comment]
It's so weird why it hasn't been updated yet , we have to close it and reapply

**Comment by tanjian917** [comment]
checker:manualTrigger

**Comment by tanjian917** [comment]
checker:manualTrigger

**Comment by tanjian917** [comment]
checker:manualTrigger

**Comment by nj-steve** [comment]
checker:manualTrigger

**Comment by nj-steve** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacebvi6gfhpbbslq6nwt4p75jbywpywmosboyed26gu4xyjdpc6ukpk 
#### Address 
> f1zvurht6mq5qbo6gq7r7wxisw25z262ocmbg7pva
#### Datacap Allocated
> 512.00TiB
#### Signer Address
> f1xx6555qijma7igpnjspyvdunc4vfxkawnpqy5ii
#### Id
> 79c2a989-b482-404a-bb8d-c48060f96afb
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacebvi6gfhpbbslq6nwt4p75jbywpywmosboyed26gu4xyjdpc6ukpk

**Comment by nj-steve** [comment]
The reports looks so nice.Willing to support it

**Comment by igoovo** [comment]
checker:manualTrigger

**Comment by igoovo** [comment]
<img width="976" alt="image" src="https://github.com/filecoin-project/filecoin-plus-large-datasets/assets/103082537/d00db38f-d428-4b48-b801-89f024a9fcba">

There are two SPs with a retrieval rate of 0. Please provide an explanation for this situation and propose improvement plans.

**Comment by tanjian917** [comment]
@igoovo SP's machines have meet some problems and being repaired. We are helping them repair them. If they still cannot repair them within 10 days, we will no longer cooperate with them.

**Comment by igoovo** [comment]
Review the history of LDN Github.
The retrieval rate is gradually improving, and the Client has made improvement plans for some SPs with low retrieval rates.
KYC is also verified as required.
There is no CID sharing, and the SPs are relatively scattered.
So willing to support the LDN in this round and will keep an eye on it.

> 
> ⚠️ All retrieval success ratios are below 1%.
> Overall Graphsync retrieval success rate: 0.00%
> ——————
> Retrieval Statistics
> Overall Graphsync retrieval success rate: 33.33%
> Overall HTTP retrieval success rate: 0.00%
> ——————
> Retrieval Statistics
> Overall Graphsync retrieval success rate: 50.00%

**Comment by igoovo** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzaced56i7t4olcllcfa3ce7ihxs2a6k6mvoklqjgwam6cnc5l3btsv4m 
#### Address 
> f1zvurht6mq5qbo6gq7r7wxisw25z262ocmbg7pva
#### Datacap Allocated
> 512.00TiB
#### Signer Address
> f1shnsfayxqll77svffaxnjenms7bbbysbqcatrpy
#### Id
> 79c2a989-b482-404a-bb8d-c48060f96afb
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaced56i7t4olcllcfa3ce7ihxs2a6k6mvoklqjgwam6cnc5l3btsv4m

**Comment by tanjian917** [comment]
checker:manualTrigger

**Comment by Bitrise0111** [comment]
The report shows healthy and we'd like to give support.

**Comment by Bitrise0111** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzaceaepyn3jyhmnmbsuuzvzwzr45dydc63emgfztgit25hsal6t5otds 
#### Address 
> f1zvurht6mq5qbo6gq7r7wxisw25z262ocmbg7pva
#### Datacap Allocated
> 1.00PiB
#### Signer Address
> f1nknj7ayq4o43czrtdoauggtwl43fbqatmqis3yy
#### Id
> fb05e8f0-5ffa-496c-98a2-1e668d5263d0
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaceaepyn3jyhmnmbsuuzvzwzr45dydc63emgfztgit25hsal6t5otds

**Comment by AlanGreaterheat** [comment]
checker:manualTrigger

**Comment by AlanGreaterheat** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzacecrjuhwziauxai4stokejzswtxs6pc4hqdgdltbfrpxh2ac5b3mas 
#### Address 
> f1zvurht6mq5qbo6gq7r7wxisw25z262ocmbg7pva
#### Datacap Allocated
> 1.00PiB
#### Signer Address
> f1pnmzlxj7cfeo2v6oj5nco46hkg2l46wj7o4xxui
#### Id
> fb05e8f0-5ffa-496c-98a2-1e668d5263d0
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacecrjuhwziauxai4stokejzswtxs6pc4hqdgdltbfrpxh2ac5b3mas

**Comment by luobin544** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacec6oau36zv5j6evc6co36yvdqsv2imnx7r3h6isa2bveobqy4avwy 
#### Address 
> f1zvurht6mq5qbo6gq7r7wxisw25z262ocmbg7pva
#### Datacap Allocated
> 2.00PiB
#### Signer Address
> f1tbd632f6w62glfaf7wjpimacbnjiz26poyoes2q
#### Id
> 0b564a24-3e08-4195-81e3-70ed98d8dac9
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacec6oau36zv5j6evc6co36yvdqsv2imnx7r3h6isa2bveobqy4avwy

**Comment by mikezli** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzaceckipbuu5s4xugyraivnz2sac76zua6nylco4izhcuheaui5clqkm 
#### Address 
> f1zvurht6mq5qbo6gq7r7wxisw25z262ocmbg7pva
#### Datacap Allocated
> 2.00PiB
#### Signer Address
> f1dnb3uz7sylxk6emti3ififcvu3nlufnnsjui6ea
#### Id
> 0b564a24-3e08-4195-81e3-70ed98d8dac9
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaceckipbuu5s4xugyraivnz2sac76zua6nylco4izhcuheaui5clqkm

**Comment by 1ane-1** [comment]
checker:manualTrigger

**Comment by 1ane-1** [comment]
No serious problem, Support for this round.

**Comment by 1ane-1** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacecuyvkrfefohs55xzpapnxcrmaqntp422kidykeentuamjtoha35s 
#### Address 
> f1zvurht6mq5qbo6gq7r7wxisw25z262ocmbg7pva
#### Datacap Allocated
> 2.00PiB
#### Signer Address
> f1mdk7s2vntzm6hu35yuo6vjubtrpfnb2awhgvrri
#### Id
> 923cbd35-8322-417e-adf4-31efb026d537
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacecuyvkrfefohs55xzpapnxcrmaqntp422kidykeentuamjtoha35s

**Comment by Bitrise0111** [comment]
check report shows healthy and we'd like to give a support, there will be a continuous check.

**Comment by Bitrise0111** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzaceaoy5r2z3xktiohbtyylueob43vgjug3deheiftq45hj6vpvh7wzc 
#### Address 
> f1zvurht6mq5qbo6gq7r7wxisw25z262ocmbg7pva
#### Datacap Allocated
> 2.00PiB
#### Signer Address
> f1nknj7ayq4o43czrtdoauggtwl43fbqatmqis3yy
#### Id
> 923cbd35-8322-417e-adf4-31efb026d537
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaceaoy5r2z3xktiohbtyylueob43vgjug3deheiftq45hj6vpvh7wzc

**Comment by herrehesse** [comment]
Mostly selfdealing with VPN abuse. Why are notaries supportive of this fraud? REPORTED.

**Comment by tanjian917** [comment]
Sorry for replying to you now, I don't understand what you are expressing. as far as I know, no SP uses VPN. Please tell me which SPs use VPN and provide evidence about yourtools. We will send your evidence and tools to the SPs and let them correct.

**Comment by kevzak** [comment]
checker:manualTrigger

**Comment by kevzak** [comment]
SPs provided:

f02221112 |  | yuanxing | GuangDong 
f01920887 | | Sindurot | Australia | 
f02246008 | Tao Calculus | Hong Kong |
f02199203 | KuQi | Inner Mongolia | 
f02370792 | Liu |Star Alliance | GuangDong | 

SPs taking deals:

f02363300 | Tokyo, Tokyo, JPAmazon.com, Inc. | 180.41 TiB | 4.09% | 180.41 TiB | 0.00%
f02519843 | Beijing, Beijing, CNChina Mobile Communicaitons Corporation | 62.31 TiB | 1.41% | 62.31 TiB | 0.00%
f02105219 | Guangzhou, Guangdong, CNChina Mobile communications corporation | 184.44 TiB | 4.18% | 184.44 TiB | 0.00%
f02760664new | Beijing, Beijing, CNChina Mobile Communications Group Co., Ltd. | 301.06 TiB | 6.82% | 301.06 TiB | 0.00%
f02199203 | Beijing, Beijing, CNCHINA UNICOM China169 Backbone | 299.91 TiB | 6.79% | 299.91 TiB | 0.00%
f02221113 | Shenzhen, Guangdong, CNCHINANET-BACKBONE | 584.81 TiB | 13.25% | 541.25 TiB | 7.45%
f02370792 | Shenzhen, Guangdong, CNCHINANET-BACKBONE | 477.56 TiB | 10.82% | 363.06 TiB | 23.98%
f02221112 | Shenzhen, Guangdong, CNCHINANET-BACKBONE | 100.00 TiB | 2.27% | 100.00 TiB | 0.00%
f02200472 | Chengdu, Sichuan, CNCHINANET-BACKBONE | 49.97 TiB | 1.13% | 49.97 TiB | 0.00%
f02363305 | Shenzhen, Guangdong, CNCTGNet | 310.75 TiB | 7.04% | 310.75 TiB | 0.00%
f02823562 | Hong Kong, Central and Western, HKHK Broadband Network Ltd. | 380.66 TiB | 8.62% | 380.66 TiB | 0.00%
f02812504new | Hong Kong, Central and Western, HKHK Broadband Network Ltd. | 272.72 TiB | 6.18% | 263.34 TiB | 3.44%
f02246008 | Hong Kong, Central and Western, HKHKBN Enterprise Solutions HK Limited | 84.53 TiB | 1.91% | 84.53 TiB | 0.00%
f02824533new | Hong Kong, Central and Western, HKHKBN Enterprise Solutions HK Limited | 68.81 TiB | 1.56% | 68.81 TiB | 0.00%
f02820089new | Hong Kong, Central and Western, HKHong Kong Beecloud System Technology Services Limited | 108.22 TiB | 2.45% | 108.22 TiB | 0.00%
f01920887 | Melbourne, Victoria, AUVocus Connect International Backbone | 25.00 TiB | 0.57% | 25.00 TiB | 0.00%
f02816095 | Singapore, Singapore, SGZenlayer Inc | 473.53 TiB | 10.73% | 473.53 TiB | 0.00%
f02816081 | Singapore, Singapore, SGZenlayer Inc | 449.66 TiB | 10.19% | 449.66 TiB | 0.00%

SPs not matching, closing until clarified

**Comment by tanjian917** [comment]
@kevzak Hello, we have cooperated with all the SPs we reported. we hope that the data can be backed up more, we subsequently cooperated with some SPs. Now, we have disclosed all SPs.
f02200472	Sichuan longcould       f02519843 	 DaoRan
f02221112 	yuanxing             f02221113   	   yuanxing 
f01920887 	 Sindurot           f02246008	  Tao Calculus
f02823562	Tao Calculus     f02363300	   xingxing
f02820089	xingxing            f02370792	  Star Alliance
f02812504 	Tao Calculus     f02105219    	Fourteen financial management
f02824533	Tao Calculus     f02199203   	KuQi 
f02760664 	KuQi           f02816095    KRAL
f02816081	KRAL         f02363305	      DaoRan
